.env.test

```diff
@@ -28,7 +28,6 @@ MONGODB_CONNECTION_STRING="mongodb://localhost:27017"
 
 
 # Ethereum Settlement
-DEFAULT_SETTLEMENT_CLIENT_RPC="http://localhost:3000"
```

Comment:
It is valid to remove this, since the `rpc_url` is being sent by anvil as a param to the child methods